src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -2814,10 +2814,13 @@ See the accompanying LICENSE file for applicable license.
         </xsl:call-template>
       </span>
       <xsl:text> </xsl:text>
-      <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-startprop ')]/revprop" mode="ditaval-outputflag"/>
-      <xsl:apply-templates/>
-      <!-- Normal end flags and revision end flags both go out after the content. -->
-      <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-endprop ')]" mode="out-of-line"/>
+      <span class="note__body">
```

Comment:
A note can have all sorts of block elements - won't this end up with paragraphs/divs/lists inside of `<span>`?

Reply:
@robander - I believe I confirmed that a `<span>` containing block elements becomes inconsequential and is handled by CSS layout engines as expected, but let me confirm and reply back here.

Reply:
I expect it would display correctly, as browsers are very forgiving. My concern is more that it's invalid HTML5 (spans allow only [phrase-level content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span)), and I don't think we should knowingly generate HTML5 that will break the spec. 

_Note ... I say knowingly because I would not be surprised to find out we have invalid HTML somewhere else. But I don't think we should set out to do that._

I'd be happy with this if it used a `<div>` rather than span, just because that would give us the container for styling but leave us with valid HTML. That said, we would need to put in some default CSS for that, unless we are setting out to change the current style where text-only notes appear directly after the **Note:** generated text.